### PR TITLE
web/portal: Login impersonate custom Action

### PR DESCRIPTION
**Pull request by fmendozzac**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Login impersonate custom Action

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/platform/src/components/Login.tsx [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { useEffect } from 'react';
 import { useStoreActions, useStoreState } from 'store';
 import { EntityValidator } from '@irontec/ivoz-ui/entities/EntityInterface';
 import { Login as DefaultLogin } from '@irontec/ivoz-ui/components';
+import queryString from 'query-string';
```
Comment (mmadariaga) [inline web/portal/platform/src/components/Login.tsx:5]: Revert this file, platform is the highest level portal. No one can exchange token here

File: web/portal/platform/src/entities/Brand/Action/Impersonate.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  if (!row) {
+    return null;
+  }
+
+  const { id } = row;
+  const queryString = `target=${id}&token=${token}`;
+
+  return (
+    <a href={`/client/?${queryString}`} target="_impersonate-client">
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Action/Impersonate.tsx:22]:
```suggestion
    <a href={`/brand/?${queryString}`} target="_impersonate-brand">
```

File: web/portal/platform/src/translations/ca.json [hunk 3]
```diff
@@ -1,1054 +1,1055 @@
 {
-    "%s emulated": "%s emulat",
```
Comment (mmadariaga) [inline web/portal/platform/src/translations/ca.json:2]: Why is this part of this PR?

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/platform/src/translations/ca.json (hunk 3 above)
Comment (fmendozzac) [reply web/portal/platform/src/translations/ca.json:2]: Added thid row:
471 - "Impersonate as brand admin": "Impersonarse como admin de marca",

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**